README.md

```diff
@@ -1,7 +1,7 @@
 # React-Redux BookStore app
 The calculator counts with:
 
-![screenshot](./src/imgs/calculator.jpg)
+![screenshot](./src/assets/imgs/Reactapp.jpg)
 
 [Live version](https://react-calculator99.herokuapp.com)
```

Comment:
- [ ] The link in your readme redirects to a calculator, let's ensure the app is available online and add the link to the readme.